RemoteIDModule/parameters.cpp

```diff
@@ -37,6 +37,7 @@ const Parameters::Param Parameters::params[] = {
     { "PUBLIC_KEY4",       Parameters::ParamType::CHAR64, (const void*)&g.public_keys[3], },
     { "PUBLIC_KEY5",       Parameters::ParamType::CHAR64, (const void*)&g.public_keys[4], },
     { "MAVLINK_SYSID",     Parameters::ParamType::UINT8,  (const void*)&g.mavlink_sysid,    0, 0, 254 },
+    { "REGION",            Parameters::ParamType::UINT8,  (const void*)&g.region,    0, 0, 3 }, // 0 = world, 1 = USA, 2 = Japan, 3 = EU
```

Comment:
should use an enum for this

Reply:
implemented an enum, but it does not compile, need help from @tridge. The enum is not known in the RemoteIDModule.ino. The code also needs a rebase.